src/blockchain/esplora/mod.rs

```diff
@@ -149,19 +149,61 @@ impl_error!(std::num::ParseIntError, Parsing, EsploraError);
 impl_error!(consensus::encode::Error, BitcoinEncoding, EsploraError);
 impl_error!(bitcoin::hashes::hex::Error, Hex, EsploraError);
 
-#[cfg(test)]
-#[cfg(feature = "test-esplora")]
-crate::bdk_blockchain_tests! {
-    fn test_instance(test_client: &TestClient) -> EsploraBlockchain {
-        EsploraBlockchain::new(&format!("http://{}",test_client.electrsd.esplora_url.as_ref().unwrap()), 20)
-    }
-}
-
 const DEFAULT_CONCURRENT_REQUESTS: u8 = 4;
 
+#[cfg(feature = "test-esplora")]
 #[cfg(test)]
-mod test {
+pub mod test {
```

Comment:
This seems like an accidental change to `pub`. Let me know if I'm mistaken.

If the idea here is to make `test::init_blockchain` externally-accessible, we need to make `init_blockchain` `pub` as well.

Reply:
Huh.. Ya this should not be public.. We don't wanna publish `init_blockhain` because user would write that on their own for translating their custom blockchain..